.changeset/light-fireants-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
```

Comment:
Isn't this a breaking change?

Reply:
In a sense but still not - nobody will have been allowed to pass in a config that matched the code. Essentially, this is probably a pretty unused feature...

I can switch to minor though, since the standalone class is exported too.

Reply:
fix'd